Functions/fetchData.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:io';
+import 'dart:convert';
+
+Map<int,Map<String,dynamic>> readingFile() {
+  
+  Map<int,Map<String,dynamic>> data = {};
+
+  print(".............Geting old record.......");
+ for(int i=0; i<2;i++){
+   print(".");
+   sleep(Duration(seconds:1));
+ }
+
+  try {
+  var path = 'dataBaseUser.txt';
+  File file = new File(path);
```

Comment:
path could have directly been given here, rather than using 2 lines for the same thing